File: Laba1/Laba1/klukva.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laba1
{
    class klukva
    {
        private int weight = 0;
        private int has_sredded = 0;

        public int Weight
        {
            set { if (value > 0) { weight = value; } }
            get { return weight; }
        }
        public int Has_sredded
        {
            set { if (value > -1 && value < 11) { has_sredded = value; } }
            get { return has_sredded; }
        }
    }
}

File: Laba1/src/main/Klukva.java
package main;

public class Klukva {

    private int weight=0;

    private int has_sredded=0;

    public int getWeight() {
        return weight;
    }

    public void setWeight(int weight) {
        if(weight>0){
            this.weight = weight;
        }

    }

    public int getHas_sredded() {
        return has_sredded;
    }

    public void setHas_sredded(int has_sredded) {
        if(has_sredded>-1 && has_sredded<11) {
            this.has_sredded = has_sredded;
        }
    }
}
